src/networking.c

```diff
@@ -2412,7 +2414,7 @@ int handleClientsWithPendingWrites(void) {
         if (!clientHasPendingReplies(c)) continue;
 
         /* If we can send the client to the I/O thread, let it handle the write. */
-        if (trySendWriteToIOThreads(c) == C_OK) continue;
+        if (try_send_to_io_threads && trySendWriteToIOThreads(c) == C_OK) continue;
```

Comment:
This variable name and the function name are very similar, but they're not the same.

In the function name we actually "try" to send. In the variable name, it's a boolean so maybe something like "should_try" is better. (Or "avoid" or "skip" if we negate the logic.)